### PR TITLE
F 908 write migration for adding polygon tokens

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related to #908  #911 

Screen Shot 1401-12-25 at 14 24 00

### CODE REVIEW

File: migration/1678950910841-insert_polygon_givback_eligible_tokens.ts [hunk 1]
```diff
@@ -0,0 +1,306 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { Token } from '../src/entities/token';
+import { NETWORK_IDS } from '../src/provider';
+
+const polygonTokens = [
```
Comment (mohammadranjbarz) [inline migration/1678950910841-insert_polygon_givback_eligible_tokens.ts:5]: @divine-comedian  I would appreciate if you take a look on tokens and addresses, and about `mainnetAddress` we need it for:

>Some tokens like PAN, XNODE, CRV dont have price on coingecko for gnosis network, So frontend guys suggested
add  mainnetAddress field for those tokens, then client can get price of these tokens in mainnet

**Review by CarlosQ96 — APPROVED** [review]
LGTM Mohammad just let us know @divine-comedian when took a look at the addreses.

**Comment by divine-comedian** [comment]
@mohammadranjbarz 

mainnet addresses that I could find 

AAVE - 0x7fc66500c84a76ad7e9c93437bfc5ac33e2ddae9
agEUR - 0x1a7e4e63778b4f12a199c062f3efdd288afcbce8
BIFI - 0x5870700f1272a1AdbB87C3140bD770880a95e55D
DAI - 0x6b175474e89094c44da98b954eedeac495271d0f
WETH - 0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2
FRAX - 0x853d955acef822db058eb8505911ed77f175b99e
LDO - 0x5a98fcbea516cf06857215779fd812ca3bef1b32
LINK - 0x514910771af9ca656af840dff83e8264ecf986ca
MAI - 0x8d6cebd76f18e1558d4db88138e2defb3909fad6
MANA - 0x0f5d2fb29fb7d3cfee444a200298f468908cc942
QUICK - 0xd2ba23de8a19316a638dc1e7a9adda1d74233368
MATIC - 0x7d1afa7b718fb893db30a3abc0cfc608aacfebb0
SAND - 0x3845badade8e6dff049820680d1f14bd3903a5d0
TUSD - 0x0000000000085d4780b73119b644ae5ecd22b376
PAXG - 0x45804880de22913dafe09f4980848ece6ecbaf78
ANKR - 0x8290333cef9e6d528dd5618fb97a76f268f3edd4
USDC - 0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48 (DECIMALS 6)
USDT - 0xdac17f958d2ee523a2206206994597c13d831ec7
UNI - 0x1f9840a85d5af5bf1d1762f925bdaddc4201f984
stMATIC - 0x9ee91f9f426fa633d227f7a9b000e28b9dfd8599
WBTC - 0x2260fac5e5542a773aa44fbcfedf7c193bc2c599

**Comment by mateodaza** [comment]
Leaving these tokens out for now until a weird error displaying wrong prices is fixed
 
AXLUSDC
KLIMA
QI
MANA

**Pull request merged**